src/components/molecules/dossier-info/index.tsx

```diff
@@ -1,6 +1,6 @@
 'use client';
 
-import moment from 'moment';
+import { formatDistanceToNow } from 'date-fns';
```

Comment:
Please fix this on all imports of this function

```suggestion
import formatDistanceToNow from 'date-fns/formatDistanceToNow';
```